tests/runtests/cli-list-suggest-autoreporting/PURPOSE

```diff
@@ -1,3 +0,0 @@
-PURPOSE of cli-list-suggests-autoreporting
```

Comment:
It would be nice to have this in the new cli as well.

Reply:
It was more or less deliberate, because I didn’t see any value in having that.

Reply:
Do you remember why it was added to begin with?

Reply:
Right, so I found https://bugzilla.redhat.com/show_bug.cgi?id=1044424, which doesn’t seem to base the assumption on anything.

Reply:
I assume that maybe this is targeting anyone who doesn’t use GNOME or any other DE with a first-time setup thing that allows enabling automatic report?

Reply:
Would moving this nagging to the shell notification script make more sense? I.e. “ABRT has detected X new problems; also, automatic reporting is disabled”.